packages/components/pie-lottie-player/src/index.ts

```diff
@@ -0,0 +1,206 @@
+
+        if (!this.animationSrc && !this.animationData) return;
+
+        try {
+            this._animationInstance = this._lottie.loadAnimation({
+                container: this._hostElement, // the dom element that will contain the animation
+                renderer: 'canvas',
+                loop: !this.loopDisabled,
+                autoplay: !this.autoPlayDisabled,
+                animationData: this.animationData,
+                path: this.animationSrc,
+            });
+            this._animationInstance.setSpeed(this.speed);
+            this._updateDirection();
+        } catch (_error:unknown) {
+            const error = _error as Error;
+            const errorMessage = error && error.message && `Error: "${error.message}"`;
+            console.error(`PieLottiePlayer: the animation couldn't be played. ${errorMessage}`);
+        }
+    }
+
+    private _destroyAnimation (): void {
+        if (!this._animationInstance) return;
+        this._animationInstance.destroy();
+    }
+
+    // Getter and Setter for animationSrc
+    @property({ type: String, reflect: true })
+    get animationSrc () {
+        return this._animationSrc;
+    }
+
+    set animationSrc (value) {
+        this._animationSrc = value;
+        // When animationSrc changes, reset animationData to undefined
+        this._animationData = undefined;
+        this._loadAnimation();
+    }
+
+    // Getter and Setter for animationData
+    @property({ type: Object })
+    get animationData () {
+        return this._animationData;
+    }
+
+    set animationData (value) {
+        this._animationData = value;
+        // When animationData changes, reset animationSrc to an empty string
+        this._animationSrc = '';
+        this._loadAnimation();
+    }
+
+    @property({ type: Boolean })
+    get autoPlayDisabled () {
+        return this._autoPlayDisabled;
+    }
+
+    set autoPlayDisabled (value) {
+        this._autoPlayDisabled = value;
+        if (!this._animationInstance) return;
+
+        this._animationInstance.autoplay = value;
+    }
+
+    @property({ type: Boolean })
+    get loopDisabled () {
+        return this._loopDisabled;
+    }
+
+    set loopDisabled (value) {
+        this._loopDisabled = value;
+        if (!this._animationInstance) return;
+
+        this._animationInstance.loop = !value;
+
+        // Ensure to resume playing if looping was disabled
+        const shouldResumePlaying = this._loopDisabled === false && this._animationInstance.isPaused && this._animationInstance.isLoaded;
+        if (shouldResumePlaying) this.play();
+    }
+
+    @property({ type: Number, reflect: true })
+    get speed () {
+        return this._speed;
+    }
+
+    set speed (value) {
+        this._speed = value;
+        if (!this._animationInstance) return;
+
+        this._animationInstance.setSpeed(value);
+    }
+
+    @property({ type: String, reflect: true })
+    get direction ():AnimationDirection {
```

Comment:
This means we don't need the other import, and the direction type doesn't need to be exported from the defs file
```suggestion
    get direction (): LottiePlayerProps['direction'] {
```